### DISCUSSION TITLE
templaterCreateNote in new tab?

### DISCUSSION THREAD

**Discussion by muteboy** [mProjectsCode/obsidian-meta-bind-plugin#256] [Q&A] [answered]
Is there a way to make the `templaterCreateNote`  command create the new note in a new tab?

**Comment by mProjectsCode** [comment, marked as answer]
No, the Templater API function that Meta Bind is using does not expose that option.

**Reply by muteboy** [reply]
Fair enough, thanks!

**Comment by MrYod** [comment]
A workaround can be used: if you put a "workspace:split-vertical" command before the templater command, then the new note will be created in the right pane, so in a new tab.
